app/models/classic-route.js

```diff
@@ -1,12 +1,9 @@
-import { computed } from '@ember/object';
-
 export default class ClassicRoute {
   id;
   isSelected = false;
   isHighlighted = false;
   paths = [];
 
-  @computed('isSelected', 'isHighlighted')
```

Comment:
This change I don't understand.

Reply:
computed isn't needed, classic component still participate in autotracking